frontend/src/metabase/public/lib/embed.ts

```diff
@@ -29,12 +28,11 @@ function getSignedToken(
   });
 }
 
-export function getSignedPreviewUrl(
+export function getSignedPreviewUrlWithoutHash(
```

Comment:
This is only used in `frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx` so it's safe to modify the return value shape. I removed the hash part because I want the URL part before the hash to stay the same when only the hash is changed.

With the previous implementation, even when the hash is changed, the whole URL will be changed too.